sdk/log/record_test.go

```diff
@@ -556,14 +556,31 @@ func TestRecordDroppedAttributes(t *testing.T) {
 		assert.False(t, called, "non-dropped attributed logged")
 
 		r.AddAttributes(attrs...)
-		assert.Equalf(t, i-1, r.DroppedAttributes(), "%d: AddAttributes", i)
-		assert.True(t, called, "dropped attributes not logged")
+		expectedDropped := 0
```

Comment:
same comment applies to other places

```suggestion
		wantDropped := 0
```